tests/cases/crashes/issue_2487_inlined_repeater_with_init.slint

```diff
@@ -0,0 +1,48 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+// Inline components requires fixing up named references of repeated element's init code inside. For Rust/C++
+// this requires SLINT_INLINING=true to achieve test coverage.
+
+
+component Compo  {
+    callback dontcrash();
+    dontcrash() => {}
+    if (true) : Rectangle {
+        init => {
+            dontcrash();
+        }
+    }
+}
+
+export component TestCase {
+    out property <bool> ok; // don't use test as interpreter test won't send mouse click to visit tree and create repeated element
```

Comment:
I think there are ways to get the interpreter test to visit the elements. I think if it is in a layout, because it is then used to compute the preferred size or something.

Reply:
Good idea, I can try that :)